### ISSUE TITLE
Converting from ONNX results in Assertion failed: axis >= 0 && axis < nbDims

### ISSUE THREAD

**Issue by CarlPoirier** [onnx/onnx-tensorrt#690]
## Description

Hi,

I'm trying to create a TRT engine from my ONNX file which you can find [here](https://drive.google.com/file/d/1NQpubIhdoRVZqaoXJTTm8ofHRgyiP7JX/view?usp=sharing). It's been trained using the torchvision.models.detection.fasterrcnn_resnet50_fpn and then I folded the constants with polygraphy with this command:
`polygraphy surgeon sanitize raccoon_resnet50.onnx --fold-constants --output raccoon_resnet50_sanitized.onnx`

Next I have been using this command:
`trtexec.exe --onnx=models/raccoon_resnet50_sanitized.onnx --workspace=2048 --saveEngine=test.engine --shapes=input:1x3x650x417 --verbose --explicitBatch `

 When doing so, it outputs the said error:
`[05/27/2021-15:44:10] [V] [TRT] ModelImporter.cpp:103: Parsing node: Flatten_1223 [Flatten]
[05/27/2021-15:44:10] [V] [TRT] ModelImporter.cpp:119: Searching for input: 2388
[05/27/2021-15:44:10] [V] [TRT] ModelImporter.cpp:125: Flatten_1223 [Flatten] inputs: [2388 -> (1, -1)],
ERROR: onnx2trt_utils.cpp:188 In function convertAxis:
[8] Assertion failed: axis >= 0 && axis < nbDims
[05/27/2021-15:44:10] [E] Failed to parse onnx file`

Now I have been finding issues with similar error message dating back from 2019 and they either suggest installing a newer version, which I obviously have, or replacing some of the code in the model definition, but much has changed since and I can't seem to find the problematic lines. Is there anything else I can look?

Thanks,

Carl

## Environment

**TensorRT Version**: 7.2.3.4
**NVIDIA GPU**: GTX 1050 Ti
**NVIDIA Driver Version**: 461.72
**CUDA Version**: 11.1
**CUDNN Version**: 8.0
**Operating System**: Windows 10 x64
**Python Version (if applicable)**: 3.8.5
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 1.9.0.dev20210526+cu111
**Baremetal or Container (if so, version)**: baremetal, python virtual environments

## Relevant Files

The ONNX model is available [here](https://drive.google.com/file/d/1NQpubIhdoRVZqaoXJTTm8ofHRgyiP7JX/view?usp=sharing).

## Steps To Reproduce

1. Download my model file
2. execute trtexec as mentioned above

**Comment by CarlPoirier** [comment]
I as able to go over this assert by exporting the model with different parameters for the batch size and it makes more sense now. I updated the linked file. Now I get a different error which is a throw of std::out_of_range in nvonnxparser. I have updated the original post.

**Comment by CarlPoirier** [comment]
Actually, this is an issue when using opset 13 only. If I export using opset 11, then I don't get the std::out_of_range.

**Comment by kevinch-nv** [comment]
Thanks for narrowing it down to a opset 13 issue. I'll take a look to see what's going on.
